modules/network-observability-loki-install.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-loki-installation_{context}"]
+= Installing the Loki Operator
+
+.Prerequisites
+
+* Installed Loki, recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.6], which allows the ability to specify a `network` tenant for Loki. The `network` tenant uses a specific tenant mode only implemented in version 5.6 of the Loki Operator. 
+* Object storage, such as AWS S3
+* {product-title} 4.10+
+* Linux Kernel 4.18+
```

Comment:
Since the first bullet point terminates with a period, the remaining bullet points should terminate with a period too.